astro-docs/src/content/docs/technologies/dotnet/Guides/migrate-from-nx-dotnet-core.mdoc

```diff
@@ -0,0 +1,448 @@
+
+```xml
+<Project>
+  <PropertyGroup>
+    <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="8.0.0" />
+  </ItemGroup>
+</Project>
+```
+
+Update project files to remove version attributes:
+
+```xml
+<!-- Before -->
+<PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
+
+<!-- After -->
+<PackageReference Include="Newtonsoft.Json" />
+```
+
+## Testing the Migration
+
+Run these commands to verify everything works:
+
+```shell {% frame="none" %}
+# Clear cache
+nx reset
+
+# Verify project detection
+nx show projects
+
+# Test builds
+nx run-many -t build
+
+# Test affected builds
+nx affected -t build
+
+# Run tests
+nx run-many -t test
+```
+
+## Troubleshooting
+
+### Projects Not Detected
+
+**Symptom**: Some .NET projects don't appear in `nx show projects`
+
+**Solution**:
+
+- Ensure project files have `.csproj`, `.fsproj`, or `.vbproj` extensions
+- Check that the plugin is registered in `nx.json`
+- Run `nx reset` to clear the cache
+
+### Build Errors: Missing Dependencies
+
+**Symptom**: Build fails with "Could not find project reference"
+
+**Solution**:
+
+- Verify the referenced project exists in the workspace
+- Ensure build targets have `"dependsOn": ["^build"]` (plugin adds this automatically)
+- Check that project references in `.csproj` files use correct relative paths
+- Run `nx reset` to clear stale cache
+
+### Targets Not Appearing
+
+**Symptom**: Expected targets like `test` or `pack` don't show up
+
+**Solution**:
+
+- `test`: Only created for projects with `IsTestProject` property or `Microsoft.NET.Test.Sdk` package
+- `pack`: Only created for library projects (not executables)
+- `publish`: Only created for executable projects (`OutputType="Exe"`)
+- Use `nx show project my-project` to see all available targets
+
+### Custom Naming Conflicts
+
+**Symptom**: Project names don't match expectations
+
+**Solution**:
+Add `<Nx><Name></Name></Nx>ProjectName` property to your `.csproj` file:
+
+```xml
+<PropertyGroup>
+  <Nx>
+    <Name>my-custom-name</Name>
+  </Nx>
+</PropertyGroup>
+```
+
+## Next Steps
+
+After completing the migration:
+
+- Review the [incremental builds guide](./incremental-builds) to understand how `--no-dependencies` optimizes builds
+- Configure [Nx Cloud](https://nx.app) for distributed caching across your team and CI
+- Set up custom targets for `serve`, `lint`, or other specialized tasks as needed
+- Update CI/CD pipelines to ensure .NET SDK 8.0+ is installed
+
+## Additional Resources
+
+- [Nx .NET Plugin Documentation](https://nx.dev/nx-api/dotnet)
+- [Nx Conformance](https://nx.dev/concepts/conformance)
+- [Central Package Management](https://learn.microsoft.com/nuget/consume-packages/central-package-management)
+- [@nx-dotnet/core Documentation](https://www.nx-dotnet.com/)
```

Comment:
is this needed since it's a migration guide?

Reply:
Eh, may be useful to understand x-ref some of the info?